lightning/src/ln/functional_tests.rs

```diff
@@ -7519,38 +7519,83 @@ fn test_upfront_shutdown_script() {
 }
 
 #[test]
-fn test_upfront_shutdown_script_unsupport_segwit() {
-	// We test that channel is closed early
-	// if a segwit program is passed as upfront shutdown script,
-	// but the peer does not support segwit.
```

Comment:
Could we replace the high level test comment?

Reply:
The comment didn't make much sense since we couldn't close a channel that was never successfully opened. And the other tests were not similarly commented. Hoping the test names are helpful using the pattern:

`test_<script-type>_[upfront_]shutdown_script`

Let me know if you have a concrete suggestion, though.